src/charm.py

```diff
@@ -333,7 +333,7 @@ def sans(self) -> List[str]:
         return sans
 
     def _is_tls_enabled(self):
-        return bool(self.cert_handler.cert)
+        return self.cert_handler.enabled
```

Comment:
So `.enabled` only means the relation is in place. Is that the semantics we're aiming for now with the `stop()` approach?

Reply:
Yes!